templates/messageextension/csharp/default/AdapterWithErrorHandler.cs.tpl

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Builder.TraceExtensions;
+using Microsoft.Bot.Connector.Authentication;
+
+namespace {{BlazorAppServer}};
```

Comment:
Since Bot and ME don't need to be a Blazor project, so we could have a better name for this kind of placeholder